autopush/db.py

```diff
@@ -129,10 +129,10 @@ def make_rotating_tablename(prefix, delta=0, date=None):
 
 
 def create_rotating_message_table(prefix="message", read_throughput=5,
-                                  write_throughput=5, delta=0):
+                                  write_throughput=5, delta=0, date=None):
     # type: (str, int, int, int) -> Table
```

Comment:
Maybe you can make create_rotating_message_table's arguments more consistent with get_rotating_message_table's while you're here?